dev/Makefile

```diff
@@ -99,6 +99,18 @@ redeploy-signet-monitoring:
 	$(TF) apply -var="bitcoin_network=signet" \
 				-var="name_prefix=galoy-sig" -auto-approve
 
-
 fmt:
 	$(TF) fmt -recursive
+
+plan:
+	$(TF) plan -sensative -var="bitcoin_network=regtest" -var="name_prefix=galoy-dev"
```

Comment:
probably a typo, but there is also no -sensitive option for terraform plan.
did you find this command useful at all?